src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml

```diff
@@ -28,7 +28,7 @@ Rectangle {
             PropertyChanges {
                 target: root
                 opacity: 0
-                visible: false
+                restoreEntryValues: false
```

Comment:
If visible is not set to false, like in the old code, will this cause screen readers to read this content even when this is in closed state?

Reply:
For this particular instance, no because the children components containing screen-reader accessible items (eg `ConnectionInfoContent`, `ConnectionInfoError`) will not be visible, even though the root (`Rectangle`) is